tests/migration/online_migration/yast2_migration.pm

```diff
@@ -218,7 +218,14 @@ sub run {
         # gpg keys; Same with leap to sle migration, need to trust packagehub gpg key.
         if (get_var('SMT_URL') =~ /smt/) {
             assert_screen 'import-untrusted-gpg-key', 180;
-            send_key 'alt-t';
+            # for 15-SP7 yast2 migration with smt, we need to choose Again and Yes.
+            if (check_var('VERSION', '15-SP7')) {
```

Comment:
use existing function would be better, ex: is_sle('15-SP7+').

Reply:
changed and rerun [**VR**](https://openqa.suse.de/tests/16141939)

Reply:
You changed to 16-SP6+ means on SLES 15SP6 has the same dialog, need confirm.

Reply:
only `>=15-SP7`, changed the code and rerun.

Reply:
@lemon-suse now the [**latest VR**](https://openqa.suse.de/tests/16141957#step/yast2_migration/17) passed, can you please take a look again and merge it. then I can trigger it in milestone and file a bug for integrity failure. Thanks